nitro/Dockerfile

```diff
@@ -13,53 +13,60 @@
 #       Each RUN contains a bundle of steps, which reduces the cache.
 
 FROM veracruz/base:latest
-ARG USER=root
-ARG UID=0
 ARG ARCH=x86_64
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    python-setuptools \
-    python3-setuptools \
-    python3 \
-    python3-pip \
-    musl-tools \
-    ca-certificates \
-    curl \
-    lxc \
-    iptables \
-    jq \
-    iproute2 \
-    apt-transport-https \
-    gnupg \
-    lsb-release \
-    software-properties-common
-
-RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
-RUN add-apt-repository "deb https://download.docker.com/linux/ubuntu bionic stable"
-RUN apt-get update && apt-get install --no-install-recommends -y docker-ce-cli
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+        apt-transport-https \
+        ca-certificates \
+        gnupg \
+        iproute2 \
+        iptables \
+        jq \
+        lsb-release \
+        lxc \
+        musl-tools \
+        python3 \
+        python3-pip \
+        python3-setuptools \
+        software-properties-common ; \
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - ; \
+    add-apt-repository "deb https://download.docker.com/linux/ubuntu bionic stable" ; \
+    apt-get update && apt-get install --no-install-recommends -y docker-ce-cli ; \
```

Comment:
Should these two lines be combined with the above `apt-get update` so we only have to do one update?

Reply:
If I remember correctly, the first `apt-get update` / `apt-get install` installs `add-apt-repository` that is used to add the docker repository...